### DISCUSSION TITLE
Unable to change the height of "FluentTextArea"

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/fluentui-blazor#237] [Q&A] [answered]
Why does changing "height" and "Rows" not change the height of "FluentTextArea"?

**Comment by vnbaaij** [comment, marked as answer]
This is a bug in the underlying library. It is raised thee already. Once fixed, it will be ported here as well,

**Reply by ghost** [reply]
Thank you!!

**Comment by LuohuaRain** [comment]
I have tried to use various methods to change the height of "FluentTextArea"... No wonder!
